src/sentry/static/sentry/app/components/events/interfaces/frame.jsx

```diff
@@ -460,13 +462,15 @@ export const Frame = createReactClass({
             <span className="symbol">
               <FunctionName frame={data} />{' '}
               {data.filename && (
-                <span
-                  className="filename"
-                  title={data.absPath !== data.filename ? data.absPath : null}
-                >
-                  {data.filename}
-                  {data.lineNo ? ':' + data.lineNo : ''}
-                </span>
+                <Tooltip title={data.absPath} disabled={!defined(data.absPath)}>
```

Comment:
Should this condition be `!defined(data.absPath) || data.absPath === data.filename`? The old title logic would hide the title if the path and absPath were the same value.